index.html

```diff
@@ -3173,13 +3163,19 @@ <h3><code>LinkedResource</code> Definition</h3>
 						<tr>
 							<td>
 								<code>
-									<dfn>alternative</dfn>
+									<dfn>alternate</dfn>
 								</code>
 							</td>
-							<td>A reference to a reformulation of the resource in an alternative format. OPTIONAL</td>
-							<td>One or more valid URL strings&#160;[[!url]].</td>
 							<td>
-								<a href="#value-array">Array</a> of <a href="#value-url">URLs</a>
+								<p>References to one or more reformulation(s) of the resource in alternative formats.</p>
+								<p>When specified, <code>encodingFormat</code> indicates the format of the reformulation.</p>
+								<p>OPTIONAL</p>
+							</td>
+							<td>
+								<p>One or more instances of <a href="#publication-link-def"><code>LinkedResource</code></a> objects.</p>
+							</td>
+							<td>
```

Comment:
In other, similar terms (e.g., resource links) the definition allows the usage of single strings instead of a full-blown Linked Resource if the target is an HTML file. See the definition of resource list (see, e.g., 2.7.3.2). I believe that, for consistency's sake, the same should apply here.

If this is indeed so, then the step 5.d in the processing steps (section 5) should also be amended, because this property is not a 'resource categorization property', but the same processing should also apply.

Reply:
Agree

Reply:
I am just knit-picking here:-)

In of 5.d there is a link to the resource categorization properties; I guess, for consistency, there should be a link to where the `alternate` is defined, too...

Reply:
Agree. In general, there aren't links to properties which are at the same level as `alternate`, so I didn't introduce this, but there are a few spots where I would think it would make sense.

Reply:
Well, one spot, anyway. The link has been added.